tests/automountlocation/test_automountlocation.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Test automountlocation
+  hosts: ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: ensure location TestLocation is absent
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: TestLocation
+      state: absent
+
+  - name: ensure location TestLocation is created
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: TestLocation
+      state: present
+
+  - name: ensure location TestLocation has been created
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: TestLocation
+      state: present
+    register: result
+    failed_when: result.changed
+
+  - name: ensure location TestLocation is absent
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: TestLocation
+      state: absent
+
+  - name: ensure location TestLocation is absent
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: TestLocation
+      state: absent
+    register: result
+    failed_when: result.changed
+
```

Comment:
Once attribute `name` is supprted as a `list` it would be nice to test against an empty list:
```yaml
  - name: ensure multiple automountlocations are absent
    ipaautomountlocation:
      ipaadmin_password: SomeADMINpassword
      name:
      - location_01
      - location_02
      state: absent
    register: result
    failed_when: not result.changed or result.failed
```